src/BasketBundle/Controller/BasketController.php

```diff
@@ -35,6 +35,19 @@
  */
 class BasketController extends Controller
 {
+    /**
+     * We inject alias 'sonata.basket.factory'.
+     */
+    private $basketFactory;
+
+    /**
+     * @param type $basketFactory
```

Comment:
`type` is not a valid class name or variable type

Reply:
I will remove entirely le hint, i can't use a valid typehint because we inject the alias 'sonata.basket.factory', who can be 'sonata.basket.session.factory' or 'sonata.basket.entity.factory', it's fine for you?